packages/ember-templates/lib/compat.js

```diff
@@ -1,7 +1,46 @@
 import Ember from 'ember-metal/core'; // reexports
 import template from './template';
+import {
+  SafeString,
+  escapeExpression,
+  htmlSafe,
+  isHTMLSafe
+} from './string';
+import EmberStringUtil from 'ember-runtime/system/string';
+import isEnabled from 'ember-metal/features';
+import { deprecate } from 'ember-metal/debug';
 
-let EmberHandlebars = Ember.Handlebars = Ember.Handlebars || {};
-let EmberHTMLBars = Ember.HTMLBars = Ember.HTMLBars || {};
 
+export let EmberHandlebars = Ember.Handlebars = Ember.Handlebars || {};
+export let EmberHTMLBars = Ember.HTMLBars = Ember.HTMLBars || {};
+let EmberHTMLBarsUtils = EmberHTMLBars.Utils || {};
+let EmberHandleBarsUtils = EmberHandlebars.Utils || {};
+
+if (isEnabled('ember-string-ishtmlsafe')) {
+  Object.defineProperty(EmberHandlebars, 'SafeString', {
+    get() {
+      deprecate(
+        'Ember.Handlebars.SafeString is deprecated in favor of Ember.String.htmlSafe',
+        false,
+        {
+          id: 'ember-htmlbars.ember-handlebars-safestring',
+          until: '3.0.0',
+          url: 'http://emberjs.com/deprecations/v2.x#toc_use-ember-string-htmlsafe-over-ember-handlebars-safestring'
+        }
+      );
+
+      return SafeString;
+    }
+  });
+} else {
+  EmberHandlebars.SafeString = SafeString;
+}
+
+EmberHTMLBars.SafeString = SafeString;
 EmberHTMLBars.template = EmberHandlebars.template = template;
+EmberHTMLBarsUtils.escapeExpression = EmberHandleBarsUtils.escapeExpression = escapeExpression;
```

Comment:
I do not believe that we ever exposed this previously on `Ember.HTMLBars`, can you confirm and if we haven't fixup?